expenses/management/commands/make_data.py

```diff
@@ -32,19 +32,35 @@ class Command(BaseCommand):
     help = "Adds demo data to database."
 
     def add_arguments(self, parser):
-        parser.add_argument('n', type=int)
+        parser.add_argument('-n', '--number', type=int)
+        parser.add_argument('--clear', action='store_true', help='Remove all data', default=False)
 
     def handle(self, *args, **options):
-        n = options['n']
+        if options['clear']:
+            User.objects.all().delete()
+            models.Expense.objects.all().delete()
+            models.Category.objects.all().delete()
+            print("Removed all data")
+            exit(0)
```

Comment:
instead of exiting, continue.  (and if you will need to exit,  use `return` instead)